src/components/Menu.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './menu.css';
+import Logo from './Logo';
+
+const Menu = () => {
```

Comment:
Nicely done with the mobile menu! I wasn't able to make it display as normal when not in mobile view so I did another menu that I didn't display in mobile view and vice versa.